pkg/network/tracer/tracer.go

```diff
@@ -437,6 +440,9 @@ func (t *Tracer) GetActiveConnections(clientID string) (*network.Connections, er
 	delta := t.state.GetDelta(clientID, latestTime, active, t.reverseDNS.GetDNSStats(), t.usmMonitor.GetHTTPStats(), t.usmMonitor.GetHTTP2Stats(), t.usmMonitor.GetKafkaStats())
 	t.activeBuffer.Reset()
 
+	clientTag := "client_id:" + strconv.FormatInt(clientID, 10)
+	tracerTelemetry.payloadSizePerClient.Set(float64(len(delta.Conns)), []string{clientTag})
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It may be just me but when I read _payload size_, I expect to see the size in terms of _bytes_. So may be something like `payload_conn_count`would be clear. But if the current name is clear to others then feel free to ignore the suggestion.

Reply:
+1